bluebottle/analytics/tasks.py

```diff
@@ -41,8 +41,13 @@ def _process_handler(backend, timestamp, tags, fields):
 
 @shared_task
 def generate_engagement_metrics():
+    # Generate metrics every 30 mins. The timestamp will be of the end date and hence in the future.
```

Comment:
@ashutoshb is this description correct? Isn't this generating metrics for the day so far?